src/Bridge/Symfony/Resources/config/crud.yaml

```diff
@@ -10,6 +10,9 @@ services:
             $formFactory: '@form.factory'
             $filters: !tagged melodiia.crud_filter
 
+    melodiia.crud.pagination.pages_request_factory:
+        class:
```

Comment:
Something is missing over here :) ... And obviously, the change of the definition of the crud is missing too.

Reply:
what do you mean by `change of the definition of the crud is missing too`

Reply:
The current configuration is not complete according to your work:

```yaml
    melodiia.crud.controller.get_all:
        class: Biig\Melodiia\Crud\Controller\GetAll
        arguments:
            $dataStore: '@melodiia.doctrine.data_provider'
            $checker: '@security.authorization_checker'
            $collectionFactory: '@melodiia.crud.filters.filter_collection_factory'
        tags: [ controller.service_arguments ]
```

Reply:
It's ok now, good catch 👍